configuration/init/systemd/tedge-mapper-aws@.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=tedge-mapper-aws checks Thin Edge JSON measurements and forwards to AWS IoT Hub.
```

Comment:
```suggestion
Description=tedge-mapper-aws checks Thin Edge JSON measurements and forwards to AWS IoT Core.
```
Since you're touching this file anyway.

Reply:
Resolved in ac91b0192809bc0880a763d83f6c87810d34eedd

Reply:
@albinsuresh Looks like we're been calling it "AWS IoT Hub" for a while :D (in the tedge-mapper-aws.service)...so good to fix it finally.